pymeasure/experiment/parameters.py

```diff
@@ -115,6 +122,20 @@ def convert(self, value):
 
         return value
 
+    def _cli_help_fields(self):
+        message = self.name
+        for field in self._help_fields:
+            if isinstance(field, str):
+                field = ["{} is".format(field), field]
```

Comment:
I´d suggest using f-strings
```suggestion
                field = [f"{field} is", field]
```

Reply:
I agree! Since `.format()` was predominantly used in the codebase, I looked for consistency.

Reply:
Ah yeah fair enough; most of those are just there out of legacy I guess.